lib/util.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+module.exports.envToJson = function (prop) {
+  var returnJSON = {};
+  var originalValue = process.env[prop];
+  if (originalValue === undefined) {
+    return {
+      'json': {},
+      'reset': function () {
+      }
+    };
+  }
+  try {
+    var grabJSON = JSON.parse(process.env[prop]);
+    returnJSON[prop] = grabJSON;
+    delete process.env[prop];
+  } catch (err) {
+    //noop
+  }
+  return {
+    'json': returnJSON,
+    'reset': function () {
+      process.env[prop] = originalValue;
+    }
+  };
+};
```

Comment:
Can you include this new directory in `Gruntfile.js` so that files underneath can be found by jshint?